azure-communication-ui/calling/src/androidTest/java/com/azure/android/communication/mocking/TestCallingSDK.kt

```diff
@@ -327,6 +327,12 @@ internal class TestCallingSDK(private val callEvents: CallEvents, coroutineConte
         return completedNullFuture()
     }
 
+    override fun startAudio() {
```

Comment:
May want more descriptive names, i.e.

startIncomingAndOutgoingAudio
stopIncomingAndOutgoingAudio

May be easy for a developer to misinterpret as single-duplex instead of full-duplex (i.e. one direction, vs bi-directional).